code/modules/mob/language/serbian.dm

```diff
@@ -302,7 +305,7 @@
 "Jerkov",
 "Jigovic",
 "Jorgic",
-"Karad�ic",
+"Karad�ic",
```

Comment:
whats with breaking simbles?

Reply:
Guessing something to do with saving it on Visual Studio? Let me fix that.